projects/neoforge/build.gradle

```diff
@@ -283,6 +283,13 @@ generateAccessTransformers {
             classesWithSuperclass('net/minecraft/world/entity/npc/VillagerTrades$ItemListing'),
             named('<init>')
     )
+
+    // Make SplashManager.SPLASHES_LOCATION public so we can pull splashes from it
```

Comment:
If it's a single entry, add it to the `accesstransformers.cfg` file under META-INF.